### DISCUSSION TITLE
Can I knock out a sprite by another sprite?

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7659] [Q&A]
![image](https://user-images.githubusercontent.com/87828192/127777338-e2dae389-2799-49ad-8517-183d7b5540b5.png)

This is from Adobe Illustrator.
I want to do kind of "path finder - minus" feature of Illustrator in the between pixi sprites.
the "mask" property doesn't work in this case.
Can I do it?

**Comment by ivanpopelyshev** [comment]
yes, make graphics rectangle , add text with blendMode=ERASE and put AlphaFilter somewhere above them to simulate the layering
